pennylane/labs/dla/dense_util.py

```diff
@@ -277,6 +358,22 @@ def trace_inner_product(
     raise NotImplementedError
 
 
+def change_basis_ad_rep(adj: np.ndarray, basis_change: np.ndarray):
```

Comment:
Do we provide the derivation somewhere, maybe in one of the following PRs? :grimacing:

Reply:
I added some explanation of _what_ the function is doing, we can add the _how_ later or feel free to add it yourself!